systemd/tests/common.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
```

Comment:
Do we need this line?

Reply:
No, that looks like a bad copy/paste from my side, removing, thanks